server/embedded/src/org/labkey/embedded/LabKeyServer.java

```diff
@@ -156,7 +162,7 @@ protected TomcatWebServer getTomcatWebServer(Tomcat tomcat)
                     }
 
                     // set the root path to the context explicitly
-                    context.setPath("");
+                    context.setPath(contextProperties.getContextPath());
```

Comment:
Any need to do trimming on this value and the legacyContextPath value? Perhaps in the setter?